lib/tether/toolbox.go

```diff
@@ -219,25 +219,30 @@ func (t *Toolbox) halt() error {
 
 // externalIP attempts to find an external IP to be reported as the guest IP
 func (t *Toolbox) externalIP() string {
-	netif, err := net.InterfaceByName("client")
+	l, err := netlink.LinkByName("client")
```

Comment:
@dougm @hickeng moved code out of toolbox. Will have a follow up PR to make `"client"` a constant.